### ISSUE TITLE
should not depend on Code::TidyAll and others

### ISSUE THREAD

**Issue by chorny** [LibreCat/Catmandu#262]
Testing with Code::TidyAll/Test::Code::TidyAll/Test::Code::TidyAll/Perl::Tidy is useful for author, but not for users - it does not test for real errors.

**Comment by jorol** [comment]
Yes, so these test are only for releases. What would the right way to define these dependencies? Like this?

```
...
on 'develop', sub {
  requires 'Code::TidyAll', 0;
  requires 'Perl::Tidy', 0;
  requires 'Test::Code::TidyAll', '0.20';
  requires 'Text::Diff', 0; # undeclared Test::Code::TidyAll plugin dependency
  };
...
```

**Comment by chorny** [comment]
Yes,'develop' stage is for such dependencies.

**Comment by jorol** [comment]
Thanks, we will change that.

**Comment by jorol** [comment]
Fixed in #263.

**Comment by nics** [comment]
thanks @jorol
